api/src/main/java/org/openmrs/validator/VisitValidator.java

```diff
@@ -159,6 +167,50 @@ private void validateStopDatetime(Visit visit, Visit otherVisit, Errors errors)
 			errors.rejectValue("stopDatetime", "Visit.visitCannotContainAnotherVisitOfTheSamePatient", messageBuilder
 			        .toString());
 		}
-		
+
+	}
+
+	private void validateStartDateTimeToBirthdate(Visit visit, Errors errors) {
+
+		Patient patient = visit.getPatient();
+
+		if (patient != null) {
```

Comment:
@dkayiwa Are there any visits without patients? Should this not fail with an error?